.pre-commit-config.yaml

```diff
@@ -0,0 +1,92 @@
+repos:
+- repo: local
+  hooks:
+    - id: docformatter
+      name: Format docstrings with `docformatter`
+      language: system
+      types: [python]
+      require_serial: true
+      entry:  docformatter
+      args: [
+        --in-place, --wrap-summaries=88, --wrap-descriptions=87, --recursive
+      ]
+      files: ^(melloddy_predictor|tests|examples)/
+-   repo: local
+    hooks:
+    - id: isort
+      name: Sorting imports with `isort`
+      language: system
+      types: [python]
+      require_serial: true
+      entry:  isort
+      files: ^(melloddy_predictor|tests|examples)/
+-   repo: local
+    hooks:
+    - id: black
+      name: Formatting files with `black`
+      language: system
+      types: [python]
+      require_serial: true
+      entry:  black
+      args: [
+        --preview,
+        --line-length=120,
+      ]
+      files: ^(melloddy_predictor|tests|examples)/
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Except for this one, all other `repo` entries target `local`, can we group the hooks instead?